docs/local_test.md

```diff
@@ -0,0 +1,51 @@
+# Test Locally
+
```

Comment:
I would add a warning that this test is for those who has access to helm-charts-priv, it will be a stop sign for others